tests/foreman/destructive/test_clone.py

```diff
@@ -118,6 +110,20 @@ def test_positive_clone_backup(target_sat, sat_ready_rhel, backup_type, skip_pul
     cloned_sat = Satellite(sat_ready_rhel.hostname)
     assert cloned_sat.cli.Health.check().status == 0
 
+    # If --skip-pulp-data make sure you can rsync /var/lib/pulp over per BZ#2013776
+    if skip_pulp:
+        # Copying satellite pulp data to target RHEL
+        assert sat_ready_rhel.execute('mkdir -p /var/lib/pulp').status == 0
+        assert (
+            target_sat.execute(
+                f'''sshpass -p "{SSH_PASS}" rsync --archive --partial --progress --compress
+                /var/lib/pulp root@{sat_ready_rhel.hostname}:/var/lib/'''
+            ).status
+            == 0
+        )
+
+    assert len(sat_ready_rhel.execute('ls /var/lib/pulp/media/exports').stdout()) > 0
```

Comment:
could this assertion be made more accurate by checking the output of the source and using that as the comparison instead of 0?

Reply:
Sure I could record the length of that before we do the backup and make sure it's the same at the end. I'm also open to other assertions that find "All the pulp data that was there before is still there". This was just a quick / easy way for me to see if the data was still there.